Templates/Java/models_extensions/BaseEntity.java.tt

```diff
@@ -7,11 +7,11 @@
 <#=host.CreatePackageDefForEntity()#>
 
 import com.google.gson.JsonObject;
-import com.google.gson.JsonElement;
 import com.google.gson.annotations.SerializedName;
 import com.google.gson.annotations.Expose;
+<# if(((OdcmClass)c).IsOpen && !((OdcmClass)c).Properties.Any()) { #>
```

Comment:
nit: Looks like this is needed for AdditionalData. Consider leaving a comment on the relation between the import of HashMap and the condition.

Reply:
it's not for additional data but for open type with no properties (that are effectively hashmaps), things like PlannerChecklistItems for instance